### ISSUE TITLE
Committees: Add 1st and 2nd vice chairs as office titles

### ISSUE THREAD

**Issue by shrayshray** [datamade/la-metro-councilmatic#639]
This year, the [Executive Management Committee](https://boardagendas.metro.net/committee/executive-management-committee-f8ac8ee68d22/) has both a 1st Vice Chair (Solis) and a 2nd Vice Chair (Najarian), instead of just Vice Chair. 1st Vice Chair and 2nd Vice Chair were recently added as board office titles for the Regular Board, could you also add them to committees?

**Comment by hancush** [comment]
Heya, @shrayshray! We already have logic in place to handle this. Specifically, committee members should be displayed in this order:

- Chair
- Vice Chair
- 1st Vice Chair
- 2nd Vice Chair
- Member

Folks holding the same role will be ordered by last name. I'll keep this open so we can double check that the new memberships are displayed as expected, once they're added! 

(Convenience link to EMC memberships in Legistar, for my reference: https://metro.legistar.com/DepartmentDetail.aspx?ID=28542&GUID=1913C09E-B178-4D3C-8844-B55F7ED448BD&R=9999a14e-e587-4b74-b8b5-05f5f824d735)

**Comment by shrayshray** [comment]
Well, this was another "doh" moment of forgetting to check "show on web" for each member, hence them not showing on the Legistar page. However, the current members *are* showing on [boardagendas.metro.net](https://boardagendas.metro.net/committee/executive-management-committee-f8ac8ee68d22/), just without Ara Najarian's 2nd Vice Chair designation (doh 2.0, Solis was entered as "Vice Chair" not 1st Vice. Corrected now):
![image](https://user-images.githubusercontent.com/28063988/89354895-e1b98580-d66e-11ea-850b-5c801a1a8925.png)
They all have "show on web" checked now. But not having it checked might have impacted how the titles showed? Or does 2nd Vice showing depend on a 1st Vice being assigned?

**Comment by hancush** [comment]
Phew, this has been a journey! The _application_ is set up to handle 1st / 2nd Vice Chair roles for committees, but the _scraper_ was not! I submitted a patch, then re-ran the person scrape to grab the correct memberships. (I also needed to remove the erroneous Vice Chair and Member roles for Najarian and Solis. To my eye, the EMC page now looks correct! How about you, @shrayshray? (And are there other committees with the 1st / 2nd Vice Chair roles?)
